applications/mlflow/DEVELOPMENT.md

```diff
@@ -105,57 +176,56 @@ This workflow tests the full Replicated release and distribution process:
    export REPLICATED_CHANNEL=channel_name
    
    # Login to the registry
-   task login:registry
+   task registry:login
```

Comment:
Is this specifically for the SDK?
I'm not a fan of the workflow needing a registry login, although I could see if you have private charts in a private registry already you could need it. Maybe this needs to explicitly be 'replicated:registry:login' or something? Given you have to log-out as part of the current workflow (before packaging) do we even want the user to have a login option or will we just do that as part of the testing process?